sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/aio/_text_analytics_client_async.py

```diff
@@ -299,6 +299,10 @@ async def recognize_pii_entities(  # type: ignore
             I.e., if set to 'phi', will only return entities in the Protected Healthcare Information domain.
             See https://aka.ms/tanerpii for more information.
         :paramtype domain_filter: str or ~azure.ai.textanalytics.PiiEntityDomainType
+        :keyword categories_filter: A list of specific PII entity categories to return.  If the value of `domain_filter`
```

Comment:
Idt it's worth mentioning `domain_filter` + `categories_filter`, I had suggested the following docstring, what do you think?

```
:keyword categories_filter: Instead of filtering over all PII entity categories, you can pass in a list of the specific PII entity categories you want to filter out. For example, if you only want to filter out U.S. social security numbers in a document, you can pass in `[PiiEntityCategoryType. US_SOCIAL_SECURITY_NUMBER]` for this kwarg.
```

Reply:
Updated!